.clj-kondo/hooks/common.clj

```diff
@@ -141,6 +141,84 @@
                 body))]
     {:node node*}))
 
+(defn with-five-bindings
```

Comment:
maybe it's worth investing time to write a macro `with-n-bindings` ? 

not a blocker though.

Reply:
How would that look in .clj-kondo/config.edn? All hooks must be functions of one argument.

Are you suggesting calling a macro in hooks.common to define all these functions for us? In that case, perhaps it would help decrease the code size a little. 

Anyway I would be very surprised to see breaking changes in how clj-kondo hooks must be defined, so I can't imagine they'd ever need to change. So the benefits of reuse vs what I've done here (essentially copy+paste) are insignificant imo. There's virtually no cost to maintaining this code